Sources/RichTextKit/Coordinator/RichTextCoordinator+Subscriptions.swift

```diff
@@ -61,17 +81,16 @@ private extension RichTextCoordinator {
         }
     }
 
-    func subscribeToTriggerAction() {
-        richTextContext.$triggerAction
+    func subscribeToIsEditingText() {
+        richTextContext.$isEditingText
             .sink(
                 receiveCompletion: { _ in },
                 receiveValue: { [weak self] in
-                    self?.handle($0)
+                    self?.setIsEditing(to: $0)
                 })
             .store(in: &cancellables)
```

Comment:
Since we're not interested in the `completion`, we can use just `sink(receiveValue:)` and rewrite this to 
```swift
richTextContext.$isEditingText.sink { [weak self] in ...
```